app/page.tsx

```diff
@@ -1,45 +1,43 @@
-"use client"
+"use client";
 
 import * as React from "react";
+import * as Toast from "@/components/toast/sender";
 
 import { BaseButton } from "@/components/button/BaseButton";
 import { InfoCircleIcon } from "@/components/icon/base/InfoCircleIcon";
 import { PageHeader } from "@/components/page/PageHeader";
-import { ErrorToast, InfoToast, SuccessToast, ToastRoot } from "@/components/toast/Toast";
 import { ThemeSwitch } from "@/components/theme/ThemeSwitch";
 
 export default function Home() {
-  const err = ErrorToast();
-  const inf = InfoToast();
-  const suc = SuccessToast();
-
   return (
     <>
       <PageHeader titl="Hello World" />
 
       <ThemeSwitch />
 
-      <BaseButton onClick={() => {
-        err.new("something's wrong");
-      }}>
+      <BaseButton
+        onClick={() => {
+          Toast.Error("Hey buddy, fix it. It broke!");
+        }}
```

Comment:
This is how a new toast can be created now. There is no need to use any react hook or component rendering here anymore, which makes the whole process simpler and easier to maintain.